packages/openbridge-webcomponents/src/components/alert-subsystem-counter/alert-subsystem-counter.css

```
@@ -0,0 +1,73 @@
+  width: 24px;
+  height: 24px;
+  color: var(--on-indent-neutral-color);
+}
+
+.icon.no-icon {
+  display: none;
+}
+
+.label {
+  @mixin font-body;
+  flex: 1 0 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)